### DISCUSSION TITLE
Diferença de medidas de SVG gerado no A Illustrator e corte

### DISCUSSION THREAD

**Discussion by Antenor-Cesar** [arkypita/LaserGRBL#1775] [Q&A]
Primeiro gostaria de agradecer a todos pelas ótimas orientações.
Faço quadros em marchetaria, e uso o vosso Software  para cortar as lâminas de 0,6 mm.
O trabalho requer precisão.
Uso o Adobe Illustrator para gerar os SVG.
O software parece não obedecer as dimensões do projeto.
Então descobri uma relação de correção que uso no AI, para obter a medida que desejo no corte 
Preciso multiplicar por fator de correção para obter a medida exata em mm no corte.

Gostaria de orientação de como fazer para projetar em mm e enviar para o corte e mm, sem ter que fazer correção.

Tem como me orientar o procedimento correto?
Estou errando no AI?
Ou as conversões usadas ao receber o arquivo  não estão corretas?

Nas versões anteriores projetava usando no AI, a unidade Pontos, e o software interpretava a mesma como mm, aí funcionava. Mas após a última versão algo deve ter mudado.
Caso necessitem de mais informações, como telas e arquivo de teste poderei enviar.
Grato pelas orientações.

**Comment by Antenor-Cesar** [comment]
Prezados, fiz o teste no Inkscape e funciona corretamente.
Estou fazendo novos teste no Adobe Illustrator, pois acho que é o AI que está gerando com erros.
Se alguém já passou por isso utilizando o AI, peço que me dê a dica para eu acertar.
Obrigado a todos.
Antenor

**Comment by Tribo-2dot0** [comment]
E aí colega... também faço marchetaria cortando lâminas. Sempre usei o corel e arquivos .plt, agora estou usando o Inkscape com .svg e da tudo certo tbm... não estou muito acostumado com o AI, mas no que eu puder ajudar pode contar.

**Comment by Antenor-Cesar** [comment]
Ok, obrigado pela resposta.
Por preferência e facilidade uso o AI, pois faço as correções em imagens no Photoshop e transporto para o AI de maneira fácil. No AI posso vetorizar ou criar diretamente, pois já estou acostumado.
Já tentei utilizar o Inkscape, apesar de funcionar bem, mas não me adaptei.
Ví que ele mantem as proporções na hora do corte.
Ao abrir um arquivo gerado no AI no Inkspace também observo diferença nas medidas.
O problema deve ser nas conversões de unidades no AI mesmo.
Ao projetar no AI, uso como unidade "Pontos" e ele reflete corretamente em "mm" na hora do corte, assim tenho mantido este método e dado certo.
Também estou a disposição.
Abraço.
Antenor.
